index.html

```diff
@@ -305,23 +305,22 @@ <h3>
           "RESOURCE-TIMING-2#dom-performanceresourcetiming">PerformanceResourceTiming</dfn></code>
           interface:
         </p>
-        <ul data-dfn-for="PerformanceResourceTiming">
-          <li>The <dfn>initiatorType</dfn> attribute MUST return the
-          {{DOMString}} "<code>navigation</code>".
-          </li>
+        <ul data-dfn-for="PerformanceNavigationTiming">
           <li>The <dfn id='dom-PerformanceNavigationTiming-workerStart'><code>
-            workerStart</code></dfn> attribute MUST return the time immediately
-            before the user agent <a data-cite=
-            "service-workers#run-service-worker">ran the worker</a> (if the
-            <a>current document</a> has an <a data-cite=
-            "service-workers#dfn-containing-service-worker-registration">active
-            service worker registration</a> [[SERVICE-WORKERS]]) required to
-            service the request, or if the worker was already available, the
-            time immediately before the user agent <a data-cite=
-            "service-workers#fetchevent">fired an event named `fetch`</a> at
-            the <a data-cite="service-workers#dfn-active-worker">active
-            worker</a>. Otherwise, if there is no active worker this attribute
-            MUST return zero.
+            workerStart</code></dfn> getter steps are to perform the following steps:
+            <ol>
+              <li>Let |workerTiming| be |this|'s [=service worker timing=].
+              <li>If |workerTiming| is null, then return 0.
+              <li>Return |workerTiming|'s [=service worker timing info/start time=].
+            </ol>
+          </li>
+          <li>The <dfn id='dom-PerformanceNavigationTiming-fetchStart'><code>
+            fetchStart</code></dfn> getter steps are to perform the following steps:
+            <ol>
+              <li>Let |workerTiming| be |this|'s [=service worker timing=].
+              <li>If |workerTiming| is null, then return |this|'s {{PerformanceResourceTiming/fetchStart}}.
```

Comment:
@yoavweiss for some reason this link references back to PerformanceNavigationTiming/fetchStart instead of referencing the superclass. Need some ReSpec foo?

Reply:
@marcoscaceres - halp?

Reply:
So, ReSpec was trying to do the right because the spec is committing a layering violation... this spec shouldn't be monkey patching `fetchStart`, which is not something this spec own.

Reply:
So Navigation Timing is inheriting from Resource Timing, and then overrides the fetchStart definition. What's the correct way to do that?

Reply:
@marcoscaceres ping? :) ^^^
What's the way in ReSpec to do an "override and call the superclass's method"?

Reply:
Solved it for now by using `data-cite`. @yoavweiss is that OK with you? I think this is reviewable now.

Reply:
Sorry I missed this! The usage is still incorrect tho :(